### PR TITLE
Robustify replicator bootup

### PR DESCRIPTION
**Pull request by ds300**
- read the initial data in a transaction, at the same time write a unique id for the boot sequence to the db
- wait for the boot id update to come through before handling messages on the subscription queue.
- stream the initial data into sqlite, rather than loading it all into memory (which caps out at 130mb)
- use the zero schema format to define data types, and to provide runtime info to generate the initial load sql query.
- added a postgres config that you can opt in to, to log out the queries that are being executed.

TODO: 

- [ ] load test the db up to 1gb of data, figure out how to measure the data size, get a sense of how many users we can theoretically support with this setup.

### Change type

- [x] `other`

### CODE REVIEW

File: apps/dotcom/sync-worker/src/TLPostgresReplicator.ts [hunk 1]
```diff
@@ -61,36 +65,83 @@ export class TLPostgresReplicator extends DurableObject<Environment> {
...
-								users.forEach((user) => this.insertRow(user, 'user'))
-								files.forEach((file) => this.insertRow(file, 'file'))
-								fileStates.forEach((fileState) => this.insertRow(fileState, 'file_state'))
+								// ok
+								await this.db.begin(async (db) => {
+									return db
+										.unsafe(sql, [], { prepare: false })
+										.simple()
+										.forEach((row: any) => {
+											this.insertRow(
```
Comment (MitjaBezensek) [inline apps/dotcom/sync-worker/src/TLPostgresReplicator.ts:122]: Might make sense to bulk insert here. We can do some tests when we'll test perf of other parts.

**Review by MitjaBezensek — APPROVED** [review]
👏

File: apps/dotcom/sync-worker/src/TLPostgresReplicator.ts (hunk 1 above)
Comment (ds300) [reply apps/dotcom/sync-worker/src/TLPostgresReplicator.ts:122]: yeah let's wait, sqlite writes are supposed to be basically instant on DOs

**Comment by ds300** [comment]
merging this now, gonna follow up with more robustness stuff before scalability testing

**Pull request merged**